ui/components/multichain/select-action-modal/__snapshots__/select-action-modal.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Select Action Modal should render correctly 1`] = `<div />`;
```

Comment:
Does this have value?  Always weird to see these single-element items.

Reply:
Have updated all the tests here https://github.com/MetaMask/metamask-extension/pull/20559/commits/a7c06b77e424302f1b3a163acc24bbe58a98f2fb#diff-00f4fb4057993bb04176ff446374b1babacc3a46177c66053e675095dfa41792R1-R4